ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -24,18 +25,20 @@ controlPlane:
     aws:
       zones:
```

Comment:
i think we can use single line array list to make it easier to set zones value easily
```
$ZONE =  '[ "zone-1",  "zone-2" ]'


aws:
  zones: $ZONES
```

Reply:
agree